frontend/src/index.css

```
@@ -1,19 +1,26 @@
 @import "./styles/tokens.css";
 @import "./styles/animations.css";
-
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
+@config "../tailwind.config.js";
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@config`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)